x-pack/plugins/cases/public/components/use_push_to_service/index.test.tsx

```diff
@@ -460,7 +438,7 @@ describe('usePushToService', () => {
       const { result } = renderHook<
         React.PropsWithChildren<UsePushToService>,
         ReturnUsePushToService
-      >(() => usePushToService({ ...defaultArgs, caseStatus: CaseStatuses.closed }), {
+      >(() => usePushToService({ ...defaultArgs, isValidConnector: false }), {
```

Comment:
`'hasErrorMessages=true if there are error messages'` so is `isValidConnector: false` just a different way of throwing an error?

Reply:
Yes, I thought I can replace the caseStatus: CaseStatuses.closed with isValidConnector: false just to be able to throw an error in this case.